src/geo/ui/widgets/widget-view.js

```diff
@@ -3,26 +3,42 @@ var WidgetLoaderView = require('./standard/widget_loader_view');
 var WidgetErrorView = require('./standard/widget_error_view');
 
 /**
- *  Default widget view:
+ * @classdesc Default widget view:
  *
- *  It contains:
- *  - view model (viewModel)
- *  - data model (dataModel)
+ * It contains:
+ * - view model (viewModel)
+ * - data model (dataModel)
  *
  *  It will offet to the user:
  *  - get current data (getData)
  *  - Sync or unsync widget (sync/unsync), making the proper view
  *  listen or not changes from the current datasource.
+ *
+ * @class geo.ui.widgets.WidgetView
```

Comment:
I don't agree that this object definition belongs in the source file itself. I think the namespace usage in e.g. core/view above makes sense and that's what I'd expect here too, and that the top-level file cartodb.js references the source to point to where the source can be found.

As a reference, I'd think what [Atom](https://atom.io/docs) is quite exemplary. They have all top-level definitions in one place and reference the source files where necessary, see [AtomEnvironment `atom`](https://atom.io/docs/api/v1.2.4/AtomEnvironment) (which would correspond to our `cdb`/`cartodb` object) vs. what they call an "essential" model (which corresponds to a specific source file) e.g. [`atom.workspace`](https://atom.io/docs/api/v1.2.4/AtomEnvironment#instance-workspace) links to [`Workspace`](https://atom.io/docs/api/v1.2.4/Workspace).